aoai_vs_oai_diffs.md

```diff
@@ -0,0 +1,81 @@
+## Diff between OAI and AOAI
+
+The differences detailed in this document were derived from comparing `{project_root}/.typespec/openapi.yaml` (OAI swagger file) and `Azure/azure-rest-api-spec/specifications/cognitiveservices/data-plane/AzureOpenAI/preview/2024-05-01-preview/inference.yaml` (AOAI latest release version for the SDKs)
+
+## Operations
+
+Listing differences in the request/response models between Azure and non-Azure
+
+### `/completions`
+
+- [x] `CreateCompletionRequest`
+  - [x] Additional Azure field: `completion_config`
```

Comment:
Some of these diffs (`completion_config`, `task` in `audio`, etc.) are, I believe, undocumented; just noting here that we should follow up on each of these to see if it's intended (in which case we should document them) or lingering old stuff (in which case we should remove them).